css/style.css

```diff
@@ -16,11 +16,6 @@ h2 {
   padding: 0;
 }
 
-h3 {
-  margin: 0;
-  padding: 0;
-}
-
 p {
```

Comment:
- [ ] Please do not delete his specification. It is important to normalize your page